tests/test_time.py

```diff
@@ -0,0 +1,159 @@
+from celest.time import Time
+from celest import units as u
+from unittest import TestCase
+import julian as jd
+import unittest
+import numpy as np
+
+
+class TestTime(TestCase):
+
+    def setUp(self):
+        self.julian = [2455368.75, 2459450.85, 2456293.5416666665]
+        self.astropy_time = time.Time(self.julian, format="jd")
+
+    def test_julian(self):
+        actual_julian = Time(self.julian).julian.to(u.jd2000)
+        self.assertTrue(np.array_equal(self.julian, actual_julian))
+
+    def test_true_solar_time_for_validation(self):
+        """Test `Time.true_solar_time`.
+
+        Notes
+        -----
+        Test cases are generated from the Global Monitoring Laboratory and the
+        Government of Australia.[1]_[2]_
+
+        References
+        ----------
+        .. [1] NOAA US Department of Commerce. ESRL Global Monitoring
+           Laboratory -Global Radiation and Aerosols. url:
+           https://gml.noaa.gov/grad/solcalc/.
+        .. [2] Time Conventions. url:
+           http://www.bom.gov.au/climate/data-services/solar/content/data-time.html.
+        """
+
+        julian = [2455368.75, 2459450.85, 2456293.5416666665]
+        longitude = [-105, -118.24, 147.46]
+
+        expected_true_solar_time = np.array([23.0715, 0.378059, 10.76556])
+        actual_true_solar_time = Time(julian).true_solar_time(longitude).to(u.hourangle)
+
+        self.assertTrue(np.allclose(expected_true_solar_time,
+                                    actual_true_solar_time, atol=0.11))
+
+    def test_mean_solar_time_for_validation(self):
+        """Test `Time.mean_solar_time`.
+
+        Notes
+        -----
+        Test cases are generated from the Government of Australia.[1]_
+
+        References
+        ----------
+        .. [1] Time Conventions. url:
+           http://www.bom.gov.au/climate/data-services/solar/content/data-time.html.
+        """
+
+        julian = 2456293.5416666665
+        longitude = 147.46
+
+        expected_mean_solar_time = np.array([10.83056])
+        actual_mean_solar_time = Time(julian).mean_solar_time(longitude).to(u.hourangle)
+
+        self.assertTrue(np.allclose(expected_mean_solar_time,
+                                    actual_mean_solar_time, atol=0.001))
+
+    def test_true_hour_angle_for_validation(self):
+        """Test `Time.true_hour_angle`.
+
+        Notes
+        -----
+        Test cases are taken from Global Monitoring Laboratory.[1]_
+
+        References
+        ----------
+        .. [1] NOAA US Department of Commerce. ESRL Global Monitoring
+           Laboratory -Global Radiation and Aerosols. url:
+           https://gml.noaa.gov/grad/solcalc/.
+        """
+
+        julian = [2455368.75, 2459450.85, 2456293.5416666665]
+        longitude = [-105, -118.24, 147.46]
+
+        expected_true_hour_angle = np.array([166.0734, 185.671, 341.53]) / 15 % 24
+        actual_true_hour_angle = Time(julian).true_hour_angle(longitude).to(u.hourangle)
```

Comment:
Using `.to(u.deg)` would allow us to remove the `/ 15 % 24` from the previous line.